webapp/app/forms/steps/lotse/personal_data.py

```diff
@@ -523,4 +523,136 @@ def render(self):
                                props=props_dict,
                                form=self.render_info.form,
                                header_title=_('form.lotse.header-title'))
-        
\ No newline at end of file
+
+class PersonAHasDisabilityPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepPersonAHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+
+    @root_validator(skip_on_failure=True)
+    def person_b_has_disability(cls, values):
+        if values.get('person_a_has_disability') != 'yes':
+            raise ValidationError
+        return values
+        
+class PersonBHasDisabilityPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepPersonBHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+
+    @root_validator(skip_on_failure=True)
+    def person_b_has_disability(cls, values):
+        if values.get('person_b_has_disability') != 'yes':
+            raise ValidationError
+        return values
+    
+class StepPauschbetragPersonA(LotseFormSteuerlotseStep):
```

Comment:
Personally, I would prefer if we could move everything related to the StepPauschbetrag to `../steps/lotse/pauschbetrag.py`. I know that we used to do this differently but there is no longer a reason to have sections of steps (e.g. personal data) and I think that having separate files & test files for each steps makes it a lot easier to find steps. WDYT?

Reply:
I like!